Tasks/c0_fib.py

```diff
@@ -5,7 +5,15 @@ def fib_recursive(n: int) -> int:
 	:param n: number of item
 	:return: Fibonacci number
 	"""
-	return None
+	if n < 0 or type(n) == float:
+		raise ValueError
```

Comment:
```suggestion
		raise ValueError("Функция определена только для натуральных чисел")
```
Ну, раз уж выбрасываете исключение, пишите в аргументах причину :)

Reply:
Неблокирующий